.github/workflows/generateAutoConfigs.yaml

```diff
@@ -0,0 +1,127 @@
+name: Generate Spring Auto-Configurations
+
+on:
+  workflow_dispatch:
+    inputs:
+      google_cloud_java_commitish:
```

Comment:
Spring Cloud GCP already depends on Libraries BOM. Can this action infer the rest of the information it needs from that instead of requiring a (potentially wrong or incompatible) google-cloud-java tag?

Reply:
Yes, we could query the pom by extracting the libraries version and using it, for example, in `https://raw.githubusercontent.com/googleapis/java-cloud-bom/libraries-bom-v26.4.0/google-cloud-bom/pom.xml -> <dependencies> -> <artifactId>gapic-libraries-bom</artifactId>`
I will add this version extraction logic to the workflow

Reply:
Added `google-cloud-java` tag inference logic